packages/2018-disaster-resilience/src/state/proactive-planning/data-parser.js

```diff
@@ -0,0 +1,23 @@
+import { civicFormat } from "@hackoregon/component-library/dist/utils";
+
+const extractProperties = features =>
+  features.map(feature => feature.properties);
+
+const processFeatures = features =>
+  features.map(nbhd => ({
+    displaced_percap: parseFloat(nbhd.displaced_percap),
+    census_response_rate: (100 - parseFloat(nbhd.census_response_rate)) / 100,
+    total_population: parseFloat(nbhd.total_population),
+    quadrant: nbhd.quadrant,
+    resilienceLabel: `${civicFormat.titleCase(
+      nbhd.name
+    )} • Census Non-Response Rate`,
```

Comment:
This is not an ideal location for something that is a chart label to live. We should look out for this, and fix the underlying issue, which I think is that the chart labels aren't well set up to display category labels

Reply:
I'd love to learn more about this. In this case I was just copying over the data transformations as is so I didn't unknowingly break something.

Reply:
I'm filing an issue for this.